mlflow/models/model.py

```diff
@@ -950,6 +951,14 @@ def log_model_metrics_for_step(client, model_id, run_id, step):
             client = mlflow.MlflowClient(tracking_uri)
             if not run_id:
                 run_id = active_run.info.run_id if (active_run := mlflow.active_run()) else None
+
+            # Log logged model params and tags to the run
+            if run_id and params:
+                client.log_batch(
+                    run_id=run_id,
+                    params=[Param(k, str(v)) for k, v in params.items()] if params else [],
```

Comment:
```suggestion
                    params=[Param(k, str(v)) for k, v in params.items()],
```
Why do we need the params check again since we're already confirming its existence on L956?

Reply:
ah thanks for the catch, let's remove it.

Reply:
Seems it's not removed :D